plugin/storage/integration/badgerstore_test.go

```diff
@@ -60,9 +61,7 @@ func (s *BadgerIntegrationStorage) clear() error {
 }
 
 func (s *BadgerIntegrationStorage) cleanUp(t *testing.T) {
-	err := s.clear()
```

Comment:
clear() function should be removed?

Reply:
I'm asking if the function itself needs to be removed.

Reply:
yes, we can also remove the `clear()` function and directly call `s.factory.Close()` in the `TestBadgerStorage` function